Automattic/jetpack
# Lint `.extra.dependencies.test-only` setting

## Proposed changes:

The setting should refer to dev dependencies of the package. Verify that this is indeed the case.

Then clean up the two references to the removed idc package that were left behind when the dependencies on it were removed.

### Other information:


## Jetpack product discussion


None

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* CI happy?
* Add an `.extra.dependencies.test-only` somewhere referring to a slug that does not exist in the monorepo (e.g. revert one of the changes in this PR). Run the script, it should be reported properly.
* Add an `.extra.dependencies.test-only` somewhere referring to an existing non-dev composer dependency of the project. Run the script, it should be reported properly.
* Add an `.extra.dependencies.test-only` somewhere referring to an existing non-dev npm dependency of the project. Run the script, it should be reported properly.
* Add an `.extra.dependencies.test-only` in `packages/changelogger` referring to `js-packages/connection`. Run the script, it should be reported properly.
* Add an `.extra.dependencies.test-only` somewhere referring to a plugin slug. Run the script, it should be reported properly.
